### ISSUE TITLE
[Bug]: OBS source is broken in latest version v3.1.0

### ISSUE THREAD

**Issue by honey-pablo** [josephdadams/TallyArbiter#734]
### What happened?

Tested with Docker, Mac & PC all acting as the Tally Arbiter controller.  Issue presents across all platforms.

To recreate:
Working with 3.0.10 - OBS controlling gpo pi tally(s).
Update to 3.1.0
Admin page does not update with OBS changes to preview/prog,   gpo pi tally does not update
Roll back to 3.0.10
reverts to working as expected 

### Version

3.1.0

### Distribution

Desktop

### OS

MacOS

### TallyArbiter configuration

```javascript
{
  "security": {
    "jwt_private_key": "9NuQONh4N9ecJRRFOZ2QgGzwhlTWVWEk3gWvNCCE69AqVhtqTXsATw7TMSr6XuVgR9fxjyCxLih3PTxIrOhFfPQdsTSAILCZqANxwJop8ubuffmVWJMIhtGfhN7VTL1Zy3x6m4AkdHTGzXU4tUD7msapUDn2oYpt06koKm/IXpT+HeYIq3RfIQo7WhqfXJxqhmlWar3u/xjxfjsrRBNjNzBlh4dA6RF2R5cnaUy0TXuuzEGfuAndkH3MmtDjt0vy8igbRdLzrYH2Ao+WIJyHhnh0/i+nCWuALLS8C1f6mlG4BTQizY033QS6MaX38N/nMMkaht1dp12wlCdvjXau5g=="
  },
  "users": [
    {
      "username": "producer",
      "password": "$2a$10$Xmypqfa45q1V2SKnc83O7ODKu4gJvEDBupwgKF1m2onxjSqLpoHDK",
      "roles": "producer"
    },
    {
      "username": "admin",
      "password": "$2a$10$g41bct1ZxfqcxQu1ICyguOnTHHSyev2KLqOgODsoI3cHf/jRmewKm",
      "roles": "admin"
    }
  ],
  "cloud_destinations": [],
  "cloud_keys": [],
  "device_actions": [],
  "device_sources": [
    {
      "deviceId": "8d35a8c1",
      "sourceIdx": "0",
      "address": "paulcam1",
      "sourceId": "90664602",
      "id": "bfb18421"
    },
    {
      "deviceId": "ccdae9c1",
      "sourceIdx": "0",
      "address": "paulcam2",
      "sourceId": "90664602",
      "id": "edffaf3d"
    },
    {
      "deviceId": "496329cf",
      "sourceIdx": "0",
      "address": "paulcam3",
      "sourceId": "90664602",
      "id": "d78f6526"
    }
  ],
  "devices": [
    {
      "name": "cam1",
      "enabled": true,
      "id": "8d35a8c1"
    },
    {
      "name": "cam2",
      "enabled": true,
      "id": "ccdae9c1"
    },
    {
      "name": "cam3",
      "enabled": true,
      "id": "496329cf"
    }
  ],
  "sources": [
    {
      "data": {
        "ip": "172.17.17.99",
        "port": 4444,
        "password": "xxxxxx",
        "version": "5"
      },
      "name": "OpenBroadcaster",
      "sourceTypeId": "4eb73542",
      "reconnect": true,
      "enabled": true,
      "id": "90664602",
      "max_reconnects": 5,
      "connected": true
    }
  ],
  "tsl_clients": [],
  "tsl_clients_1secupdate": false,
  "bus_options": [
    {
      "id": "e393251c",
      "label": "Preview",
      "type": "preview",
      "color": "#3fe481",
      "priority": 50
    },
    {
      "id": "334e4eda",
      "label": "Program",
      "type": "program",
      "color": "#e43f5a",
      "priority": 200
    },
    {
      "id": "12c8d699",
      "label": "Aux 1",
      "type": "aux",
      "color": "#0000FF",
      "priority": 100
    },
    {
      "id": "12c8d689",
      "label": "Aux 2",
      "type": "aux",
      "color": "#0000FF",
      "priority": 100
    }
  ],
  "externalAddress": "http://0.0.0.0:4455/#/tally",
  "remoteErrorReporting": false,
  "uuid": "cbbfef66"
}
```

**Comment by josephdadams** [comment]
Could be related to #733 and ultimately my PR #726 , so I will look into this and push a `v3.1.1` as soon as I have time to do so.

**Comment by josephdadams** [comment]
fixed in `v3.1.1`
